### PR TITLE
Executive - 2021-11-25

### PR DESCRIPTION
**Pull request by LongForWisdom**
90% of the way there...

### CODE REVIEW

File: governance/votes/Executive vote - November 26, 2021.md [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+
+The Governance Facilitator(s) and the Protocol Engineering Core Unit have placed an executive proposal into the voting system. MKR Holders should vote for this proposal if they support the following alterations to the Maker Protocol.
+
+If you are new to voting in the Maker Protocol, please see the [voting guide](https://community-development.makerdao.com/en/learn/governance/how-voting-works/) to learn how voting works, and this [wallet setup guide](https://community-development.makerdao.com/en/learn/governance/voting-setup/) to set up your wallet to vote.
+
+---
+
+## Executive Summary
+
+If this executive proposal passes, the following **changes** and **additions** will occur within the Maker Protocol:
```
Comment (mkrorbrkr) [inline governance/votes/Executive vote - November 26, 2021.md:18]: Small thing, but the template has change and additions separately, this seems fine though tbh

File: governance/votes/Executive vote - November 26, 2021.md [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+- The **wBTC-C** vault type will be onboarded to the Maker Protocol with the parameters below.
+- The **GUSD peg stability module** will be onboarded to the Maker Protocol with the parameters below.
+- The **Surplus Buffer** will be increased $fill_in
+- The **Aave DAI direct deposit module** Debt Ceiling will be increased from 50 million DAI to 100 million DAI.
```
Comment (mkrorbrkr) [inline governance/votes/Executive vote - November 26, 2021.md:22]: I'm wondering about capitalisation for "direct deposit module" - we've capitalised it in previous execs: https://github.com/makerdao/community/blob/master/governance/votes/Executive%20vote%20-%20October%2029%2C%202021.md

File: governance/votes/Executive vote - November 26, 2021.md [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+
+If you are new to voting in the Maker Protocol, please see the [voting guide](https://community-development.makerdao.com/en/learn/governance/how-voting-works/) to learn how voting works, and this [wallet setup guide](https://community-development.makerdao.com/en/learn/governance/voting-setup/) to set up your wallet to vote.
+
+---
+
+## Executive Summary
+
+If this executive proposal passes, the following **changes** and **additions** will occur within the Maker Protocol:
+- The **wBTC-C** vault type will be onboarded to the Maker Protocol with the parameters below.
+- The **GUSD peg stability module** will be onboarded to the Maker Protocol with the parameters below.
```
Comment (mkrorbrkr) [inline governance/votes/Executive vote - November 26, 2021.md:20]: Should "peg stability module" be capitalised?

File: governance/votes/Executive vote - November 26, 2021.md [hunk 4]
```diff
@@ -0,0 +1,150 @@
...
+The Governance Facilitator(s) and the Protocol Engineering Core Unit have placed an executive proposal into the voting system. MKR Holders should vote for this proposal if they support the following alterations to the Maker Protocol.
+
+If you are new to voting in the Maker Protocol, please see the [voting guide](https://community-development.makerdao.com/en/learn/governance/how-voting-works/) to learn how voting works, and this [wallet setup guide](https://community-development.makerdao.com/en/learn/governance/voting-setup/) to set up your wallet to vote.
+
+---
+
+## Executive Summary
+
+If this executive proposal passes, the following **changes** and **additions** will occur within the Maker Protocol:
+- The **wBTC-C** vault type will be onboarded to the Maker Protocol with the parameters below.
```
Comment (mkrorbrkr) [inline governance/votes/Executive vote - November 26, 2021.md:19]: "wBTC" or "WBTC" - we've used both previously, last week's exec used "WBTC": https://github.com/makerdao/community/blob/master/governance/votes/Executive%20vote%20-%20November%2019%2C%202021.md

File: governance/votes/Executive vote - November 26, 2021.md (hunk 3 above)
Comment (mkrorbrkr) [inline governance/votes/Executive vote - November 26, 2021.md:20]: Link to the PSM docs on makerworld?

File: governance/votes/Executive vote - November 26, 2021.md [hunk 5]
```diff
@@ -0,0 +1,150 @@
...
+- The **GUSD peg stability module** will be onboarded to the Maker Protocol with the parameters below.
+- The **Surplus Buffer** will be increased $fill_in
```
Comment (mkrorbrkr) [inline governance/votes/Executive vote - November 26, 2021.md:21]: Link to the Surplus Buffer docs?

File: governance/votes/Executive vote - November 26, 2021.md [hunk 6]
```diff
@@ -0,0 +1,150 @@
...
+- The **Aave DAI direct deposit module** Debt Ceiling will be increased from 50 million DAI to 100 million DAI.
+- **906,805 DAI** will be transferred to [SAS-001](https://mips.makerdao.com/mips/details/MIP39c2SP25), [IS-001](https://mips.makerdao.com/mips/details/MIP39c2SP24) and [DECO-001](https://mips.makerdao.com/mips/details/MIP39c2SP23) as per their budget proposals.
```
Comment (mkrorbrkr) [inline governance/votes/Executive vote - November 26, 2021.md:23]: I make it 906,806 by my calculation

Comment (mkrorbrkr) [reply governance/votes/Executive vote - November 26, 2021.md:23]: Found the issue, there's a rounding error, your number is correct

File: governance/votes/Executive vote - November 26, 2021.md (hunk 2 above)
Comment (mkrorbrkr) [inline governance/votes/Executive vote - November 26, 2021.md:22]: Shouldn't it be "Maximum Debt Ceiling" as we're using an IAM?

File: governance/votes/Executive vote - November 26, 2021.md [hunk 7]
```diff
@@ -0,0 +1,150 @@
...
+### wBTC-C Vault Type Onboarding
+
+As per the successful outcome of this [governance poll](https://vote.makerdao.com/polling/QmdVYMRo?network=mainnet#poll-detail), wBTC-C will be onboarded with the parameters listed below if this executive proposal passes.
+
+**Collateral Parameters**
+
+* Underlying Collateral: Wrapped Bitcoin (WBTC)
+* [Stability Fee](https://community-development.makerdao.com/en/learn/governance/param-stability-fee): 1.5%
+* [Liquidation Ratio](https://community-development.makerdao.com/en/learn/governance/param-liquidation-ratio): 175%
+* [Debt Ceiling (`line`)](https://community-development.makerdao.com/en/learn/governance/param-debt-ceiling): 1 billion DAI
```
Comment (mkrorbrkr) [inline governance/votes/Executive vote - November 26, 2021.md:48]: Shouldn't it be "Maximum Debt Ceiling" as we're using an IAM?

File: governance/votes/Executive vote - November 26, 2021.md [hunk 8]
```diff
@@ -0,0 +1,150 @@
...
+
+\* Note that the initial Maximum Debt Ceiling is being set to 10 million DAI to limit risk. A proposal to increase the Maximum Debt Ceiling to 100 million will be included in a future executive vote.
+
+### Surplus Buffer Linear Increase
+
+As per the successful outcome of this [governance poll](https://vote.makerdao.com/polling/QmUqfZRv?network=mainnet#poll-detail), the Surplus Buffer will be increased $fill_in
+
+### Increase Aave D3M (DAI Direct Deposit Module) Debt Ceiling 
+
+As per the successful outcome of this [governance poll](https://vote.makerdao.com/polling/QmZhvNu5?network=mainnet), the Aave D3M Debt Ceiling will be increased from 50 million DAI to 100 million DAI if this executive proposal passes.
```
Comment (mkrorbrkr) [inline governance/votes/Executive vote - November 26, 2021.md:104]: Shouldn't it be "Maximum Debt Ceiling" as we're using an IAM?

Comment (mkrorbrkr) [reply governance/votes/Executive vote - November 26, 2021.md:104]: plus link to debt ceiling docs?

File: governance/votes/Executive vote - November 26, 2021.md [hunk 9]
```diff
@@ -0,0 +1,150 @@
...
+
+## Proposal Details
+
+### wBTC-C Vault Type Onboarding
+
+As per the successful outcome of this [governance poll](https://vote.makerdao.com/polling/QmdVYMRo?network=mainnet#poll-detail), wBTC-C will be onboarded with the parameters listed below if this executive proposal passes.
+
+**Collateral Parameters**
+
+* Underlying Collateral: Wrapped Bitcoin (WBTC)
```
Comment (mkrorbrkr) [inline governance/votes/Executive vote - November 26, 2021.md:45]: capitalised WBTC here, needs to be consistent whichever way you choose to go with it.

File: governance/votes/Executive vote - November 26, 2021.md (hunk 7 above)
Comment (mkrorbrkr) [reply governance/votes/Executive vote - November 26, 2021.md:48]: And should it link to the DC-IAM page as it has a different description of debt ceiling? That's what you've done for the GUSD-PSM

File: governance/votes/Executive vote - November 26, 2021.md [hunk 10]
```diff
@@ -0,0 +1,150 @@
...
+
+A total of **366,563 DAI** will be streamed to the Immunefi Security ([IS-001](https://mips.makerdao.com/mips/details/MIP39c2SP24)) Core Unit at [0xd1F2eEf8576736C1EbA36920B957cd2aF07280F4](https://etherscan.io/address/0xd1F2eEf8576736C1EbA36920B957cd2aF07280F4) over 8 months starting 2021-12-01 as per their most recent [budget proposal](https://mips.makerdao.com/mips/details/MIP40c3SP42).  
+
+A total of **424,944 DAI** will be streamed to the Content Production ([MKT-001](https://mips.makerdao.com/mips/details/MIP39c2SP5)) Core Unit at [0xDCAF2C84e1154c8DdD3203880e5db965bfF09B60](https://etherscan.io/address/0xDCAF2C84e1154c8DdD3203880e5db965bfF09B60) over 5 months starting 2021-12-01 as per their most recent [budget proposal](https://mips.makerdao.com/mips/details/MIP40c3SP33).  
+
+A total of **5,121,875 DAI** will be streamed to the Deco Fixed Rate ([DECO-001](https://mips.makerdao.com/mips/details/MIP39c2SP23)) Core Unit at [0xF482D1031E5b172D42B2DAA1b6e5Cbf6519596f7](https://etherscan.io/address/0xF482D1031E5b172D42B2DAA1b6e5Cbf6519596f7) over 33 months starting 2021-12-01 as per their most recent [budget proposal](https://mips.makerdao.com/mips/details/MIP40c3SP35).
```
Comment (mkrorbrkr) [inline governance/votes/Executive vote - November 26, 2021.md:134]: The MIP for this says to be streamed over 36 months, but the spreadsheet says 33, just want to check this is right?

**Review by mkrorbrkr — COMMENTED** [review]
made a few comments, mostly minor stuff

File: governance/votes/Executive vote - November 26, 2021.md (hunk 1 above)
Comment (LongForWisdom) [reply governance/votes/Executive vote - November 26, 2021.md:18]: Yep, was a conscious choice. Happy with this as is.

File: governance/votes/Executive vote - November 26, 2021.md (hunk 2 above)
Comment (LongForWisdom) [reply governance/votes/Executive vote - November 26, 2021.md]: Fixed

File: governance/votes/Executive vote - November 26, 2021.md (hunk 3 above)
Comment (LongForWisdom) [reply governance/votes/Executive vote - November 26, 2021.md]: Fixed

File: governance/votes/Executive vote - November 26, 2021.md (hunk 4 above)
Comment (LongForWisdom) [reply governance/votes/Executive vote - November 26, 2021.md:19]: Eh, not too worried about this one. I prefer lowercase today lol.

File: governance/votes/Executive vote - November 26, 2021.md (hunk 3 above)
Comment (LongForWisdom) [reply governance/votes/Executive vote - November 26, 2021.md]: I think I did this in second change.

File: governance/votes/Executive vote - November 26, 2021.md (hunk 2 above)
Comment (LongForWisdom) [reply governance/votes/Executive vote - November 26, 2021.md]: Eh, I'm simplifying for the summary. But yeah, I changed it.

File: governance/votes/Executive vote - November 26, 2021.md (hunk 7 above)
Comment (LongForWisdom) [reply governance/votes/Executive vote - November 26, 2021.md:48]: We more or less always have DC-IAM now. The Debt Ceiling docs are better, but yeah agree.

Comment (LongForWisdom) [reply governance/votes/Executive vote - November 26, 2021.md:48]: Have changed

File: governance/votes/Executive vote - November 26, 2021.md (hunk 8 above)
Comment (LongForWisdom) [reply governance/votes/Executive vote - November 26, 2021.md]: Yes, already fixed this one.

File: governance/votes/Executive vote - November 26, 2021.md (hunk 9 above)
Comment (LongForWisdom) [reply governance/votes/Executive vote - November 26, 2021.md:45]: Need is a strong word, and technically the token and the vault type are different.

Comment (LongForWisdom) [reply governance/votes/Executive vote - November 26, 2021.md:45]: Actually the ilk should be capitalized more than the collateral.

Comment (LongForWisdom) [reply governance/votes/Executive vote - November 26, 2021.md:45]: Ahha! Now I've swapped both of them.

File: governance/votes/Executive vote - November 26, 2021.md (hunk 10 above)
Comment (LongForWisdom) [reply governance/votes/Executive vote - November 26, 2021.md:136]: The mip also says 33 months (in the implementation section)

Comment (LongForWisdom) [reply governance/votes/Executive vote - November 26, 2021.md:136]: 3 months up-front, I think.

Comment (mkrorbrkr) [reply governance/votes/Executive vote - November 26, 2021.md:136]: I need to work on my reading comprehension

File: governance/votes/Executive vote - November 26, 2021.md [hunk 11]
```diff
@@ -0,0 +1,152 @@
...
+---
+
+## Executive Summary
+
+If this executive proposal passes, the following **changes** and **additions** will occur within the Maker Protocol:
+- The **wBTC-C** vault type will be onboarded to the Maker Protocol with the parameters below.
+- The **GUSD [Peg Stability Module](https://makerdao.world/en/learn/governance/module-psm)** will be onboarded to the Maker Protocol with the parameters below.
+- The **[System Surplus Buffer](https://makerdao.world/en/learn/governance/param-system-surplus-buffer)** will be increased to 90 million DAI over a period of 30 weeks.
+- The **Aave DAI Direct Deposit Module** [Maximum Debt Ceiling](https://makerdao.world/en/learn/governance/module-dciam) will be increased from 50 million DAI to 100 million DAI.
+- **906,805 DAI** total will be transferred to [SAS-001](https://mips.makerdao.com/mips/details/MIP39c2SP25), [IS-001](https://mips.makerdao.com/mips/details/MIP39c2SP24) and [DECO-001](https://mips.makerdao.com/mips/details/MIP39c2SP23) as per their budget proposals.
```
Comment (mkrorbrkr) [inline governance/votes/Executive vote - November 26, 2021.md:23]: I wonder if "A total of **906,805 DAI** will be transferred..." reads more clearly?

Not much in it tbf

File: governance/votes/Executive vote - November 26, 2021.md [hunk 12]
```diff
@@ -0,0 +1,152 @@
...
+
+As per the successful outcome of this [governance poll](https://vote.makerdao.com/polling/QmdVYMRo?network=mainnet#poll-detail), wBTC-C will be onboarded with the parameters listed below if this executive proposal passes.
+
+**Collateral Parameters**
+
+* Underlying Collateral: Wrapped Bitcoin (wBTC)
+* [Stability Fee](https://community-development.makerdao.com/en/learn/governance/param-stability-fee): 1.5%
+* [Liquidation Ratio](https://community-development.makerdao.com/en/learn/governance/param-liquidation-ratio): 175%
+* [Maximum Debt Ceiling (`line`)](https://makerdao.world/en/learn/governance/module-dciam): 1 billion DAI
+* [Target Available Debt (`gap`)](https://makerdao.world/en/learn/governance/module-dciam): 100 million
```
Comment (mkrorbrkr) [inline governance/votes/Executive vote - November 26, 2021.md:49]: Do we need to include that this is denominated in DAI, so "100 million DAI"?

File: governance/votes/Executive vote - November 26, 2021.md [hunk 13]
```diff
@@ -0,0 +1,152 @@
...
+* [Proposal](https://forum.makerdao.com/t/proposal-gusd-psm/11148)
+* [Risk Evaluation](https://forum.makerdao.com/t/psm-gusd-a-collateral-onboarding-risk-evaluation/11559)
+* [Technical Evaluation](https://forum.makerdao.com/t/gusd-erc20-token-smart-contract-technical-assessment/4603)
+* [Oracle Evaluation](https://forum.makerdao.com/t/mip10c3-sp13-proposal-gusd-oracle-collateral-onboarding-oracle-assessment/4777)
+
+\* Note that the initial Maximum Debt Ceiling is being set to 10 million DAI to limit risk. A proposal to increase the Maximum Debt Ceiling to 100 million will be included in a future executive vote.
+
+### Surplus Buffer Linear Increase
+
+As per the successful outcome of this [governance poll](https://vote.makerdao.com/polling/QmUqfZRv?network=mainnet#poll-detail), the [System Surplus Buffer](https://makerdao.world/en/learn/governance/param-system-surplus-buffer) will be increased from 60 million DAI to 90 million DAI over a period of 30 weeks if this executive proposal passes.
```
Comment (mkrorbrkr) [inline governance/votes/Executive vote - November 26, 2021.md:100]: Do you think this needs to be a bit clearer? So "over a period of 30 weeks, by increasing the System Surplus Buffer by 1 million DAI per week"? or something along those lines

**Review by mkrorbrkr — COMMENTED** [review]
Nearly there I think, just a couple of further small things

File: governance/votes/Executive vote - November 26, 2021.md (hunk 9 above)
Comment (mkrorbrkr) [reply governance/votes/Executive vote - November 26, 2021.md]: Yeah fair, need was a bit strong. Probably hungry and cranky when I wrote that. Sorry.

File: governance/votes/Executive vote - November 26, 2021.md (hunk 11 above)
Comment (LongForWisdom) [reply governance/votes/Executive vote - November 26, 2021.md:23]: Makes sense, changed.

File: governance/votes/Executive vote - November 26, 2021.md (hunk 12 above)
Comment (LongForWisdom) [reply governance/votes/Executive vote - November 26, 2021.md:49]: Yeah, probably. I copied that directly from the poll lol.

File: governance/votes/Executive vote - November 26, 2021.md (hunk 13 above)
Comment (LongForWisdom) [reply governance/votes/Executive vote - November 26, 2021.md:100]: Yep, worth being explicit

**Comment by mkrorbrkr** [comment]
LGTM!

**Review by mkrorbrkr — APPROVED** [review]

**Pull request merged**